TeensyDmx.cpp

```diff
@@ -236,7 +209,7 @@ void TeensyDmx::nextTx()
             m_dmxBufferIndex = 0;
         } else {
             m_uart.write(m_activeBuffer[m_dmxBufferIndex]);
-            ++m_dmxBufferIndex;
+            m_dmxBufferIndex++;
```

Comment:
I like to prefer prefix increment in my C++ code as it is more efficient for non-primitives.  It doesn't really make much difference with a decent compiler, but still not a necessary change.

Reply:
Done, and changed others in the code to be consistent.